Browser_IDE/editorMain.js

```diff
@@ -749,14 +749,22 @@ function audioFunctionNotification(source) {
 async function runProgram(){
     try {
         clearErrorLines();
-
-        displayEditorNotification(activeLanguageSetup.compiled ? "Compiling project..." : "Building project...", NotificationIcons.CONSTRUCTION);
-
+		
+		// the notification object returned by displayEditorNotification
+		let currentNotification = displayEditorNotification(
+			activeLanguageSetup.compiled ? "Compiling project..." : "Building project...",
```

Comment:
I think we could improve the readability of this line if we move the ternary out of the function call. 

For example:

```
const notificationMessage = activeLanguageSetup.compiled ? "Compiling project..." : "Building project...";
displayEditorNotification(notificationMessage, ...);
```